README.rst

```diff
@@ -12,7 +12,8 @@ This plugin is used to fast import data from `Common Crawl`_ into
 Crate_. The plugin extends a custom schema ``ccrawl`` to the copy
 from command which allows to import gzipped common crawl data to
 a table. See also our `blog post`_ about what we did here! Also, feel
-free to contribute if you spot any bugs.
+free to contribute if you spot any bugs. *This plugin requires Crate version
+0.55.0 or higher.*
```

Comment:
i would put it right under the headline in bold. this is atm the most important information.